flow/src/main/java/com/sequenceiq/flow/controller/FlowController.java

```diff
@@ -70,13 +73,15 @@ public FlowCheckResponse hasFlowRunningByFlowId(String flowId) {
 
     @Override
     @AccountIdNotNeeded
-    public Page<FlowLogResponse> getFlowLogsByFlowIds(List<String> flowIds, int size, int page) {
-        return flowService.getFlowLogsByIds(flowIds, PageRequest.of(page, size));
+    public List<FlowLogResponse> getFlowLogsByFlowIds(List<String> flowIds, int size, int page) {
+        Page<FlowLogResponse> response = flowService.getFlowLogsByIds(flowIds, PageRequest.of(page, size));
+        return isNull(response) ? newArrayList() : response.getContent();
```

Comment:
you can use `CollectionUtils.emptyIfNull(response.getContent())`

Reply:
The current logic checks if `response` is null and returns an emptyList. With `CollectionUtils.emptyIfnull(response.getContent())`, the logic changes because we are checking if `response.getContent()` is null. The UTs caught this with an NPE 😅

Reply:
i see,then this should be ok